src/Controller/Speaker/Create.php

```diff
@@ -48,7 +48,13 @@ public function handle(Request $request): Response
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            if (!empty($speakerRequest->photo)) {
+                $speakerRequest->photoPath = $this->fileUploader->upload($speakerRequest->photo);
+            }
+
+            if (empty($speakerRequest->photoPath)) {
+                $speakerRequest->photoPath = $_SERVER['HTTP_ORIGIN'].'/images/default_speaker.svg';
```

Comment:
Use `UrlGeneratorInterface` rather than `$_SERVER` variables directly.
Other solution: inject Symfony HTTP Request and `"{$request->getScheme()}://{$request->getHttpHost()}"`